### ISSUE TITLE
Ability to access iNat from Photo share options

### ISSUE THREAD

**Issue by carrieseltzer** [inaturalist/INaturalistIOS#117]
I also want to be able to select one or more photos from within iOS Photos, tap the share icon, and see the iNat icon as as an option.

Then it would open iNat, create a new observation, add the selected photos, and pop up the message about metadata.

This was one of the options that Alex laid out in a clickable prototype, but he also suggested that I add these as issues on github. I think this is closely related to https://github.com/inaturalist/INaturalistIOS/issues/116 since they both require the ability to select multiple photos.

**Comment by kueda** [comment]
Related, but probably going to involve separate coding. I'd categorize this as a nice-to-have after #46, though it might be easy to roll into that.

**Comment by carrieseltzer** [comment]
#46 appears to be mostly about how to deal with positional accuracy when selecting a photo taken outside the iNat app, but if what I've described above is a "photo share activity", then this might be a duplicate of #46.

**Comment by kueda** [comment]
Not a duplicate, just something extra to do when #46 is working.

**Comment by alexshepard** [comment]
this is in fact a dupe of #46
